tests/bindings/test.swift

```diff
@@ -26,16 +26,21 @@ do {
     let vdc = VdcCollection(engine: storage)
     let mdocPayload = try String(contentsOf: URL(fileURLWithPath: "../../../tests/res/mdoc.b64"))
     let mdocBytes = Data(base64Encoded: mdocPayload)
-    let uuid = UUID.init()
+    let uuid = UUID()
     try await vdc.add(
-        credential: Credential(
-            id: uuid.uuidString, format: CredentialFormat.msoMdoc,
-            type: CredentialType("org.iso.18013.5.1.mDL"), payload: mdocBytes!, keyAlias: "alias"))
+        credential: Credential.init(
```

Comment:
NOTE: I've added a construction method for Credential to return an `Arc<Credential>`

CC @rschulman